Lecture-4/Task4/l4task4/Runner.java

```diff
@@ -0,0 +1,20 @@
+package l4task4;
+
+public class Runner {
+
+    private final String name;
+    private final int time;
```

Comment:
Time is not a part of a runner class, like `score` is not a part of `Student` class

Reply:
done